src/status_im2/contexts/quo_preview/calendar/calendar_day.cljs

```diff
@@ -0,0 +1,45 @@
+(ns status-im2.contexts.quo-preview.calendar.calendar-day
+  (:require [status-im2.contexts.quo-preview.preview :as preview]
+            [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]
+            [quo2.core :as quo]))
+
+(def descriptor
+  [{:label   "State:"
+    :key     :state
+    :type    :select
+    :options [{:key   :default
+               :value "Default"}
+              {:key   :selected
```

Comment:
For booleans please end the variable name with '?' - it's a convention in cljs. E.g :disabled?

Reply:
but here they are not boolean, they are just different value (text) of the state (default, disabled, selected, today)

Reply:
:thinking:  this indicates the variant, which can be: `:default`, `:disabled`, `:selected` or `:today`. that will be passed to the component inside the `:state` key, so I'd say it's ok  the way they are named  :+1:  (without `?`).

But I'm not sure about the `:state` key  name :thinking:  since we usually call "state" to the component's state managed using ratoms or hooks.

wdyt?
@J-Son89 @mohsen-ghafouri @ilmotta

Reply:
yes i agree with you. following the guideline, I had to use this name.

Reply:
I think one acceptable alternative is to flatten state out.
e.g 
```
(defn view [{:keys [state]] ... )
```
where `state` is a map of `selected`, `disabled, `today` and `default` - 

A reasonable alternative is
```
(defn view [{:keys [selected? disabled? today?]] ... )
```
as the names are still taken from the Figma prop for state - mostly it's about not naming things "random" things that are different from the design system.
e.g we don't want to use something like `current-day?` instead of `today?`

Reply:
It's a reasonable suggestion @J-Son89, but I think it's leaning on the side of us trying to be smart in the quo2 code for little return. Using the same prop as in Figma helps us, for instance, create preview screens that closely match the names in Figma. If we flatten out the state, then the developer will most likely create 3 different boolean fields in the preview screen, which will a bit more confusing to the designer.

Reply:
Sure, I'm good with that 👍